tools/build_env/build_env.sh

```diff
@@ -40,18 +43,18 @@ if ! which docker > /dev/null; then
 fi
 
 if [[ $BUILD = true ]]; then
-  readonly DOCKER_ROOT="$(dirname $0)"
-  docker build --rm -t $IMAGE_NAME $DOCKER_ROOT
+  readonly DOCKER_ROOT="$(dirname $0)/$TAG"
```

Comment:
Wow, that behavior of the `latest` tag is pretty evil :-/

In any case, I think my suggestion was misunderstood. I'm suggesting to move away of using tags to differentiate the Android image from the regular image. Instead, use different image names (e.g. `uproxy/build` and `uproxy/build-android`). If you have different Dockerfiles, that means they are different images, and they should have different names.

Why do you consider different image names are less elegant?